.travis.yml

```diff
@@ -4,3 +4,6 @@ node_js:
   - 'iojs'
   - '0.12'
   - '0.10'
+before_install:
+  - "npm config set spin false"
+  - "npm install -g npm@^2"
```

Comment:
this is totally legit change. Otherwise we can't use @sindresorhus npm namespace